modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/FacetsAssert.java

```diff
@@ -43,15 +42,10 @@ public static void assertFrequencies(
 
 		Assert.assertNotNull(termCollectors);
 
-		Stream<TermCollector> stream = termCollectors.stream();
+		List<String> termCollectorStrings = TransformUtil.transform(
```

Comment:
@julschong Please inline this variable to avoid possible headache in naming... You can just `String.valueOf(TransformUtil.xxx)`.